quark/script/ciphey.py

```diff
@@ -2,14 +2,22 @@
 # This file is part of Quark-Engine - https://github.com/quark-engine/quark-engine
 # See the file 'LICENSE' for copying permission.
 
-from ciphey import decrypt
-from ciphey.iface import Config
-
 
 def checkClearText(inputString: str) -> str:
     """Check the decrypted value of the input string.
 
     :param inputString: string to be checked.
     :return: the decrypted value
     """
+    try:
+        from ciphey import decrypt
+        from ciphey.iface import Config
+
+    except ImportError:
+        raise Exception(
+            "Ciphey is not installed. Please use the command"
+            " 'python3 -m pip install ciphey --upgrade'"
+            " to install the package."
+        )
```

Comment:
Please set the cause of the exception. In this case, the cause is the exception in line 16.
You can refer [here](https://peps.python.org/pep-3134/#explicit-exception-chaining) for the best practice.